apps/test/unit/instructionsComponentsTest.js

```diff
@@ -10,73 +10,67 @@ describe('instructions components', () => {
 
   describe('MarkdownInstructions', function() {
     it('standard case had top padding and no left margin', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            markdownClassicMargins={false}
-            inTopPane={false}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          markdownClassicMargins={false}
+          inTopPane={false}
+          noInstructionsWhenCollapsed={true}
+        />
       );
-      var element = dom.find('.instructions-markdown').first();
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(19);
       expect(element.props().style.marginBottom).to.equal(35);
       expect(element.props().style.marginLeft).to.equal(undefined);
       expect(element.text()).to.equal('md');
     });
 
     it('inTopPane has no top padding', function() {
-      var dom = mount(
-        <div>
-          <StatelessMarkdownInstructions
-            markdown="md"
-            inTopPane={true}
-            noInstructionsWhenCollapsed={true}
-          />
-        </div>
+      const wrapper = mount(
+        <StatelessMarkdownInstructions
+          markdown="md"
+          inTopPane={true}
+          noInstructionsWhenCollapsed={true}
+        />
       );
-      var element = dom.find('.instructions-markdown').first();
+      const element = wrapper.find('.instructions-markdown').first();
       expect(element.props().style.paddingTop).to.equal(0);
     });
   });
 
   describe('NonMarkdownInstructions', function() {
     it('can have just instructions', function() {
-      var dom = mount(
-        <div>
-          <NonMarkdownInstructions
-            puzzleTitle="title"
-            shortInstructions="instructions"
-          />
-        </div>
+      const wrapper = mount(
+        <NonMarkdownInstructions
+          puzzleTitle="title"
+          shortInstructions="instructions"
+        />
       );
-      var elements = dom.childAt(0).children();
+      const elements = wrapper
+        .find('div')
+        .first()
+        .children();
```

Comment:
[`children()` now has a slightly different meaning](https://github.com/airbnb/enzyme/blob/master/docs/guides/migration-from-2-to-3.md#children-now-has-slightly-different-meaning). In this case, we had to update how we find the set of elements rendered by `NonMarkdownInstructions` to not include its root `div` so our tests continued to be valid.